package.json

```diff
@@ -2,8 +2,11 @@
   "name": "tube-tracker-back-end",
   "version": "1.0.0",
   "description": "",
-  "main": "app.js",
```

Comment:
I actually think we can remove this. AFAIK, this is only used if your project is being consumed as a node module.